plugins/catalog-backend/src/ingestion/processors/types.ts

```diff
@@ -76,6 +76,12 @@ export type LocationProcessor = {
   ): Promise<void>;
 };
 
+export type RelationSpec = {
+  type: string;
+  source: { kind: string; namespace?: string; name: string };
+  target: { kind: string; namespace?: string; name: string };
```

Comment:
i don't think these namespaces will be optional; use the type `EntityName` here

Reply:
We didn't have `ENTITY_DEFAULT_NAMESPACE` when I wrote this, so wanted to avoid hardcoded constants too deep into other logic